components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConsumerImpl.java

```diff
@@ -1639,6 +1639,8 @@ else if ("subcontext".equalsIgnoreCase(searchType)) {
                     criteria=APIConstants.API_OVERVIEW_CONTEXT;
                 }else if (searchType.equalsIgnoreCase("Description")) {
                     criteria=APIConstants.API_OVERVIEW_DESCRIPTION;
+                } else if (searchType.equalsIgnoreCase("Tag")) {
```

Comment:
Fixed in : https://github.com/wso2/carbon-apimgt/commit/67c064f7410655acdbfedac15735038bc54ddaa7